client/src/components/events/event.css

```diff
@@ -0,0 +1,59 @@
+.img{
```

Comment:
the rest of the codebase is using styled components. would it be possible to use styled components instead of css?
I know it's a hassle to learn a new tool when something you're familiar with already does the job, but I feel that it's pretty important to maintain consistency of the tool we're using in the codebase!